elastic/apm-server
# [cloud] Set config settings for continuous cloud sizes

## Motivation/summary

Do not restrict cloud settings to predefined sizes to support potential sizing changes in the future (for example allow larger deployments than 8GB). 
Pleateau on number of workers at `20` (aligned with current upper limit of cloud apm deployments of 8GB). 

## How to test these changes
* Start withouth `CLOUD_APM_CAPACITY`: should not set any default values for `output` and `queue` 
* Start withouth `CLOUD_APM_CAPACITY` set to one of the previously defined values (`512`, `1024`, `2048`, `4096`, `8192`) should set default values for `output` and `queue` to current hardcoded values
* Start withouth `CLOUD_APM_CAPACITY=xyz` should set default values for `output` and `queue` according to the implemented formula
## Related issues

related to #5372
follow up on https://github.com/elastic/apm-server/pull/5402
